app/controllers/api_controller.rb

```diff
@@ -1,5 +1,3 @@
-class ApiController < ApplicationController
```

Comment:
There's no need for a custom controller. If you just don't set a parent controller it should work